packages/plugin-elevenlabs/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@elizaos/plugin-elevenlabs",
+  "version": "1.0.0-alpha.0",
+  "type": "module",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": {
+      "import": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@elizaos/core": "workspace:*",
+    "tsup": "8.3.5"
+  },
+  "scripts": {
+    "build": "tsup --format esm --dts",
+    "dev": "tsup --format esm --dts --watch",
+    "test": "vitest run"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "agentConfig": {
+    "pluginType": "elizaos:plugin:1.0.0",
```

Comment:
If this is a "v2" plugin that won't be compatible with v1, any chance we can update this metadata to make that clear?

Reply:
All the V2 plugins metadata will be updated in a separate PR. 🙂 

cc @odilitime